i18n/en.json

```diff
@@ -1533,12 +1533,6 @@
   "admin.systemUserDetail.teamList.header.name": "Name",
   "admin.systemUserDetail.teamList.header.role": "Role",
   "admin.systemUserDetail.teamList.header.type": "Type",
-  "admin.systemUserDetail.teamList.teamRole.admin": "Team Admin",
```

Comment:
These translations shouldn't be removed, `team_row.tsx` is still using them.

Reply:
Yes i wanted to ask about this, leaving those makes the `i18n-check` fail, so i ran `make i18n-extract` and it removed that, so it honestly got a bit confusing.

Reply:
Hmm, looks like our `i18n-extract` script isn't taking into account `.tsx` files...that's a problem.

Reply:
So what should be the next line of action?

Reply:
I'll make a fix to the tool and we'll go from there. Sorry for the delay!

Reply:
https://github.com/mattermost/mattermost-webapp/pull/4759

Reply:
@ikeohachidi The fix should be in now. I've merged master into your branch for you, so all you need to do is pull in the latest changes, revert the `en.json` file and run `make i18n-extract` and things should be working.